src/tools/kotlin-codegen-shared.ts

```diff
@@ -31,8 +31,10 @@ export function generateConnectionSpecType(connection: HandleConnectionSpec, nod
   }
 
   return (function generateType(type: Type): string {
-    if (type.isEntity) {
-      const node = nodes.find(n => n.schema.equals(type.getEntitySchema()));
+    if (type.isEntity || type.isVariable) {
```

Comment:
@piotrswigon What do you think about handling this situation?

Reply:
👍